packages/p2p-v2/src/components/Modals/PaymentMethods/styles.scss

```diff
@@ -29,9 +29,25 @@
         gap: 1.6rem;
         width: 100%;
         &--cancel {
-            background: transparent;
             border: 2px solid #999999;
-
+            vertical-align: middle;
+            align-items: center;
+            justify-content: center;
+            cursor: pointer;
+            white-space: nowrap;
+            display: inline-flex;
+            padding: 0.5rem 1.2rem;
+            height: 4rem;
+            border-radius: 4px;
+            transition: all 0.2s cubic-bezier(0.65, 0.05, 0.36, 1);
+            &:hover:not([disabled]) {
+                background: rgba(0, 0, 0, 0.08);
+            }
+            &:disabled,
+            &[disabled] {
+                opacity: 0.32;
+                cursor: not-allowed;
+            }
```

Comment:
These seem to be more general that `p2p`, so I think they should be handled inside of `@deriv-com/ui` will tag Shayan here once I'm done with the changes here
```suggestion
            vertical-align: middle;
            align-items: center;
            justify-content: center;
            cursor: pointer;
            white-space: nowrap;
            display: inline-flex;
            padding: 0.5rem 1.2rem;
            height: 4rem;
            border-radius: 4px;
            transition: all 0.2s cubic-bezier(0.65, 0.05, 0.36, 1);
            &:hover:not([disabled]) {
                background: rgba(0, 0, 0, 0.08);
            }
            &:disabled,
            &[disabled] {
                opacity: 0.32;
                cursor: not-allowed;
            }
```